src/core/public/overlays/flyout/flyout_service.tsx

```diff
@@ -77,14 +77,7 @@ export interface OverlayFlyoutStart {
 /**
  * @public
  */
-export interface OverlayFlyoutOpenOptions {
-  className?: string;
-  closeButtonAriaLabel?: string;
-  ownFocus?: boolean;
-  'data-test-subj'?: string;
-  size?: EuiFlyoutSize;
-  maxWidth?: boolean | number | string;
-}
+export type OverlayFlyoutOpenOptions = Omit<EuiFlyoutProps, 'onClose'>;
```

Comment:
Please, put the interface back. The resulting type is not documentation-friendly.

Reply:
Ok but then you have to keep in sync with EUI. I had to this because this interface is missing some prop.

Reply:
> then you have to keep in sync with EUI

Additional props should be added on demand when necessary. We just did not add properties from the EUI types that weren't used atm. Plus, this keeps an abstraction layer between `core` and `EUI` which is just the correct practice when delegating to underlying libraries.

Reply:
OK thanks for clarifying @pgayvallet, makes sense! 👍